test/vm-customize

```diff
@@ -58,6 +60,12 @@ def run_script(machine_instance, scriptlist):
         else:
             print >> sys.stderr, "Bad path to script:", foo
 
+def upload_files(machine_instance, uploadfiles):
+    """Upload files/directories inside image"""
+    for foo in uploadfiles:
+        srcfile, dest = foo.split(":")
+        machine_instance.upload([srcfile], dest)
```

Comment:
This can lead to issues when `srcfile` is relative to a directory different from `testinfra.TEST_DIR`, since that is used in the upload function: https://github.com/cockpit-project/cockpit/blob/a3844d34943a478c3db9a475015c85b794aadb3c/test/common/testinfra.py#L42
I suggest trying something like
```
# machine_instance.upload tries to interpret relative paths in relation to testinfra.TEST_DIR
src_absolute = os.path.join(os.getcwd(), srcfile)
machine_instance.upload([src_absolute], dest)
```
here instead.